submissions/5mountains/popup/css/main.css

```diff
@@ -0,0 +1,260 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style-type: none;
+    text-decoration: none;
+    outline: none;
```

Comment:
You should be careful with resetting the `outline` property because it can be useful when you navigate your website using tabs.

Reply:
In this small project, I remove default properties and set my own) but again - I will change it!

Reply:
Ok, it's not a blocker for approving this PR. It's only a reminder for you for the future.